features/privacy-dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+    "_meta": {
+        "description": "Common UI used across DuckDuckGo Browsers and Extensions",
+        "sampleExcludeRecords": {
+            "domain": "example.com",
+            "reason": "site breakage"
+        }
+    },
+    "state": "enabled",
```

Comment:
Set this default `disabled` or just remove the key so that other platforms don't have it enabled. Even if they aren't implementing the feature its better to keep things consistent across the config.